lib/Doctrine/ORM/Query/SqlWalker.php

```diff
@@ -1831,7 +1831,7 @@ public function walkUpdateItem($updateItem)
     public function walkWhereClause($whereClause)
     {
         $condSql  = $whereClause !== null ? $this->walkConditionalExpression($whereClause->conditionalExpression) : '';
-        $discrSql = $this->generateDiscriminatorColumnConditionSql($this->rootAliases);
+        $discrSql = $this->generateDiscriminatorColumnConditionSQL($this->rootAliases);
```

Comment:
Psalm doesn't bug you about this at any level, because *sigh* PHP. Should it be something to warn a dev about?

Reply:
Well, PHP is case-insensitive when it comes to function/method/class names, but mixing them may have dire consequences (especially for classes and PSR-0/4).
I think check for case sensitive invocation is essential and should exist. But PHPStan does that on level 0 although it's not a bug and works fine for functions. Probably because this check is a side-effect of CallToNonExistentFunctionRule which checks for fatal error on level 0. I'd say it should be split and case-sensitivity check moved to different level.

Reply:
> Well, PHP is case-insensitive when it comes to function/method/class names, but mixing them may have dire consequences (especially for classes and PSR-0/4)

Definitely has dire consequences for class names (with autoloaders) but I've yet to see it have any consequences for function/method names. Still, obviously bad code.

Reply:
Yeah, this is more CS than static analysis stuff. Still good idea to keep the casing consistent 👍

Reply:
Can you CC me on any suggestions/ideas for PHPStan in the future? :) I might move the case-sensitivity check on functions/methods to `strict-rules`...